fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatisticBuilder.java

```diff
@@ -56,9 +56,24 @@ public ColumnStatisticBuilder(ColumnStatistic columnStatistic) {
         this.updatedTime = columnStatistic.updatedTime;
     }
 
-    public ColumnStatisticBuilder setCount(double count) {
+    // ATTENTION: DON'T USE FOLLOWING TWO DURING STATS DERIVING EXCEPT FOR INITIALIZATION
+    public ColumnStatisticBuilder(double count) {
```

Comment:
if count in ColumnStatistic is not used anymore, why still have a ctor with count? need count to compute datasize?

Reply:
it is used to compatible with current code in the initialization stage, but not in the deriving stage.